koku/masu/processor/expired_data_remover.py

```diff
@@ -164,5 +178,17 @@ def remove(self, simulate=False, provider_uuid=None, line_items_only=False):
                         expired_date=expiration_date, simulate=simulate
                     )
             else:
+                # Remove expired CostUsageReportManifests
+                expiration_date = self._calculate_expiration_date()
```

Comment:
The expiration date is already being calculated here: https://github.com/project-koku/koku/blob/466c5c31e6b25f23622839c78fec148ece713b5a/koku/masu/processor/expired_data_remover.py#L172

So you probably don't need to recalculate it here.

Reply:
added